[1.4] Port ItemCommand

ExterminatorX99 opened the pull request:
### What are the changes to ExampleMod?
Ported ItemCommand

### Do these changes need additional documentation?
<!-- If you feel like these examples deserve more attention, such as in a tutorial, let us know here -->

ExterminatorX99 committed:
Port ItemCommand

Steviegt6 commented on ExampleMod/Common/Commands/ItemCommand.cs:
```diff
@@ -0,0 +1,46 @@
+using Terraria;
+using Terraria.ModLoader;
+
+#nullable enable
```
Appears to be unnecessary? Doesn't matter much, but could confuse people unfamiliar with nullables.

ExterminatorX99 committed:
Remove nullable annotation

It was only there to shut up ReSharper mistakenly saying there's a possible NullReferenceException

andrewlord607 commented:
https://github.com/tModLoader/tModLoader/pull/1440 already exists? :thinking:
